### DISCUSSION TITLE
MultiSelect 在表格的编辑弹窗中无法更新选中值，无法删除等情况

### DISCUSSION THREAD

**Discussion by colinlin778** [dotnetcore/BootstrapBlazor#3492] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

勾选或取消选中，MultiSelect 的编辑框的值没有变化
关闭按钮点击时，MultiSelect 的编辑框的值没有变化
![93PB _ %Y0Z4@A$8B $KJDV](https://github.com/dotnetcore/BootstrapBlazor/assets/169628375/f518e64e-f060-4c54-8923-7df05ecc3e11)

### .NET Version

8.5.6

**Comment by bb-auto** [comment]
@colinlin778 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@colinlin778 duplicate #3456

**Reply by ArgoZhang** [reply]
@colinlin778 右上角项目帮忙点一下 star 谢谢啦
